site/src/pages/components/switch.explainer.mdx

```diff
@@ -334,15 +338,9 @@ Limitations:
 
 ### Accessibility
 
-{/* - A labelling text can be assigned to the switch like any other form control through a `<label>` element, the `aria-label=""` attribute or the `aria-labelledby=""` attribute */}
-{/* - Should be assigned the implicit aria role of "switch" */}
-{/* - The current value should be reported as "on" or "off" to screen readers */}
+The control can be in a checked or unchecked state. Exposure of states will be defined by AT. [^6]
```

Comment:
Sorry, a bit of a misunderstanding here. Browsers expose the state - screen readers decide how then to “announce” the state to their users. You can drop the second sentence here

Reply:
No worries! I was not sure how detailed this needs to be defined anyways. I'm happy to get feedback on this.